clients/devtools-entry.js

```diff
@@ -50,13 +50,12 @@ function listenForStatus(listenCallback) {
 }
 
 if (typeof module !== 'undefined' && module.exports) {
-  // export for require()ing (via browserify).
-  module.exports = {
-    runLighthouseInWorker,
-    listenForStatus,
-    registerLocaleData,
-    lookupLocale,
-  };
+  // Ideally this could be exposed via browserify's `standalone`, but it doesn't
+  // work for LH because of https://github.com/browserify/browserify/issues/968.
+  // Instead, since this file is only ever run in node for testing, expose a
+  // bundle entry point as global.
+  // @ts-ignore
+  global.runBundledLighthouse = lighthouse;
```

Comment:
originally was going to make `runLighthouseInWorker` global, but we need to substitute in a `connection` and `config` (instead of `port` and `categoryIds`). Rather than making the function unreadable with a bunch of options or requiring devtools to call the module differently (which we could still do in the future), exposing the inner `lighthouse` call seemed like the simplest fix.

I am slightly concerned about drift between how devtools calls `runLighthouseInWorker` and what happens inside that function and the default interface, but for now it's a really simple correspondence.

Reply:
This sounds good to me. When I have the smoke tests run against the Audits panel, it will be at a higher level than this bundle, so the drift between `runBundledLighthouse` and `runLighthouseInWorker` is irrelevant (won't even used the former). I will need to add a `config`, but that's simple.

So while `runBundledLighthouse` happens to live here, it's got nothing to do with devtools. That is a little confusing - what about making a new client for this, `bundle-entry.js`?

Reply:
> When I have the smoke tests run against the Audits panel, it will be at a higher level than this bundle

ah, good to know, I didn't realize that, though it makes sense.

> So while runBundledLighthouse happens to live here, it's got nothing to do with devtools. That is a little confusing - what about making a new client for this, bundle-entry.js?

Agree it's kind of confusing, but the benefit is it's getting all the bundle customizations--all the `isDevTools` tweaks, for instance--so it's making sure the real bundles are working. I was thinking we could add a smoke test just for the pubAds integration, for instance.

Originally I had it something like
```js
function runLighthouseInWorker() {
  // ...
  runLighthouse(...args);
}

/**
 * Intermediate function to expose for testing.
 */
function runLighthouse() {
  lighthouse(...args)
}

global.runBundledLighthouse = runLighthouse;
```
that spelled it out a little more clearly but also seemed a bit silly.

We could change the `runLighthouseInWorker` interface to make it more flexible, too (like make it take a config and devtools code could call `getDefaultConfigForCategories` with the `categoryIds` itself and then pass the config back in), but I figured now's probably not the time for such changes :)

Other ways we could make things more explicit and clearer?